packages/plugin-commands-installation/src/recursive.ts

```diff
@@ -180,6 +184,9 @@ export default async function recursive (
       const localConfig = await memReadLocalConfig(rootDir)
       const { manifest, writeProjectManifest } = manifestsByPath[rootDir]
       let currentInput = [...input]
+      if (updateMatch) {
+        currentInput = matchDependencies(updateMatch, unpatternedInput, manifest, include)
```

Comment:
I think it doesn't make sense to pass in the unpatternedInput. Just concat it outside

```suggestion
        currentInput = [
          ...unpatternedInput,
          ...matchDependencies(updateMatch, manifest, include),
        ]
```